components/engine_rocks/src/import.rs

```diff
@@ -28,15 +31,26 @@ impl ImportExt for RocksEngine {
                 .map_err(|e| format!("sync {}: {:?}", file, e))
                 .map_err(r2e)
         })?;
+        let now = Instant::now_coarse();
         // This is calling a specially optimized version of
         // ingest_external_file_cf. In cases where the memtable needs to be
         // flushed it avoids blocking writers while doing the flush. The unused
         // return value here just indicates whether the fallback path requiring
         // the manual memtable flush was taken.
-        let _did_nonblocking_memtable_flush = self
+        let did_nonblocking_memtable_flush = self
             .as_inner()
             .ingest_external_file_optimized(cf, &opts.0, files)
             .map_err(r2e)?;
+        let time_cost = now.saturating_elapsed_secs();
+        if did_nonblocking_memtable_flush {
+            INGEST_EXTERNAL_FILE_TIME_HISTOGRAM
+                .with_label_values(&[cf_name, "non_block"])
```

Comment:
please make it static

Reply:
Not a good choice.

The `cf_name` cannot be static as expected.